feature-libs/cart/base/components/add-to-cart/add-to-cart.component.ts

```diff
@@ -144,6 +159,19 @@ export class AddToCartComponent implements OnInit, OnDestroy {
 
     this.addToCartForm.controls['quantity'].setValue(1);
 
+    if (this.featureToggles.showRealTimeStockInPDP) {
+      this.currentProductService
+        .getRealTimeStock(this.productCode, this.sapUnit)
+        .pipe(take(1))
+        .subscribe(({ quantity, availability }) => {
+          this.maxQuantity = Number(quantity);
+          this.hasStock = Boolean(
+            availability && availability !== 'outOfStock'
+          );
+          this.cd.markForCheck();
```

Comment:
QUESTION: do we need this line?

Reply:
Yes it intentionally added to make the change take effect quicker.